code/style.css

```diff
@@ -1,4 +1,194 @@
-html {
-  background: #0025ff;
-  color: #fff;
-}
+        html {
+            position: relative;
+            min-height: 100%;
+        }
+        
+        body {
+            background: #dae6f0;
+            font-family: 'Roboto', sans-serif;
+            margin: 0;
+        }
+        /*------------ header---------------- */
```

Comment:
Really nice to see the usage of comments throughout your CSS-file, when the projects become a bit bigger it's really useful to do that 🏅🎖